base/linalg/dense.jl

```diff
@@ -486,9 +505,22 @@ If `factorize` is called on a Hermitian positive-definite matrix, for instance,
 will return a Cholesky factorization.
 
 Example:
-```julia
-A = diagm(rand(5)) + diagm(rand(4),1); #A is really bidiagonal
-factorize(A) #factorize will check to see that A is already factorized
+```jldoctest
+julia> A = Bidiagonal(ones(5, 5), true) #A is really bidiagonal
```

Comment:
Calling the `Bidiagonal` constructor to clarify what's going on is a great idea! The remainder of the example relies on `A` being an `Array` though. Perhaps wrap your `Bidiagonal` constructor call in an `Array` constructor call (`Array(Bidiagonal(ones(5, 5), true))`)?

Calling the `Bidiagonal` constructor makes this example sufficiently clear that the comment may no longer  be necessary?